README.md

```diff
@@ -29,3 +29,7 @@ This README documents steps necessary to get the application up and running.
 
 * Run project
   - Run ```rails s``` and visit [http://localhost:3000/](http://localhost:3000/) in your browser!
+
+* Tests
+ - Run (*ensure your postgreSQL is running as well*)
+    - ```rspec```
```

Comment:
- [ ] Kindly document your README file using the [Microverse README template](https://github.com/microverseinc/readme-template). Include all necessary headers and customize the content to match your project. By doing this, your README file will become more descriptive and professional. Also, note that **Hello Rails app** is not the title of this app. This is a **Blog App**

Reply:
Hi @FrederickMih , Thanks so much for your review. On ti making the requested changes right away.

Reply:
Hi @FrederickMih , Thank you so much for your review once more. I realized I was too excited that I resumed work from the initial hello test app. I have made the required changes and appreciate your professional review.

> HappyCoding